### ISSUE TITLE
NVDA is not reading names/contacts when @ mentioning someone in Word comments

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13530]
### Steps to reproduce:
Launch word and open a document uploaded in One drive.
Navigate to review tab and activate new comment button.
Navigate to comment edit field and type "@".
Observe that a group of contact list will be appeared and navigate with down arrow.
Observe the announcement of NVDA.
Note: This issue is happening with all the NVDA versions.

### Actual behavior:
When we type "@", we will get suggestions, NVDA is not announcing the contact name when we navigate through it. It is announcing as "@" for every suggestion. While navigating through the suggestions, It is announcing the content in edit field we type.

### Expected behavior:
NVDA should announce the suggestions available information and contact names while navigating through it.

#### NVDA installed/portable/running from source:
Installed & portable

#### NVDA version:
2019.3, 2021.3.4

#### Windows version:
19559.1000

#### Name and version of other software in use when reproducing the issue:
Word, 19559.1000

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, same behavior in 2019 and 2021 builds

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Manousakis01** [comment]
Hey, I would like to take this issue and try to fix it. I am new to this project but I will give my best shot. I also saw that this problem can also be reproduced in other Office apps like PowerPoint.

**Comment by feerrenrut** [comment]
@msftedad Can you confirm if this is a duplicate of your other issue #13494 ?

**Comment by Manousakis01** [comment]
@feerrenrut should I wait for @msftedad to reply? Or can I start working on this?

**Comment by Manousakis01** [comment]
@feerrenrut I am trying to resolve this issue a couple days now and found out that the @mention list is not a focusable object in Microsoft word. That's why when trying to use tab to focus it does not work. Maybe a solution could be to handle this list the same way as the cursor is on the objects since cursor hovering reads the mentioned users. What do you think about it? Do you have any suggestions?

**Comment by michaelDCurran** [comment]
Duplicate of #13494  which was fixed by pr #14222
